docs/requirements.txt

```diff
@@ -1,5 +1,5 @@
 dwave-ocean-sdk
-sphinx
+sphinx>=2.0.0
```

Comment:
We currently have for https://github.com/dwavesystems/minorminer/blob/master/docs/requirements.txt a requirement that sphinx==1.*
I think that was needed for the Python IF to build

Reply:
Note that https://github.com/dwavesystems/minorminer/issues/102 is now closed. @arcondello, any reason not to proceed with this merge?